frontend/doc-site/05__Annotate and Analyze Your Data/5_6__Community Extensions.mdx

```diff
@@ -2,28 +2,28 @@
 
 This project was started with the sole goal of empowering the scientific community to explore and understand their data. As such, we encourage other scientific tool builders in academia or industry to adopt the patterns, tools, and code from this project. All code is freely available for reuse under the [MIT license](https://github.com/chanzuckerberg/cellxgene/blob/main/LICENSE.txt).
 
-Before extending cellxgene, we encourage you to reach out to us with ideas or questions. It might be possible that an extension could be directly contributed, which would make it available for a wider audience, or that it's on our roadmap and under active development.
+Before extending CellxGene, we encourage you to reach out to us with ideas or questions. It might be possible that an extension could be directly contributed, which would make it available for a wider audience, or that it's on our roadmap and under active development.
```

Comment:
```suggestion
Before extending CELLxGENE Annotate, we encourage you to reach out to us with ideas or questions. It might be possible that an extension could be directly contributed, which would make it available for a wider audience, or that it's on our roadmap and under active development.
```

This document is about Annotate.